terraform/modules/hub/analytics.tf

```diff
@@ -0,0 +1,97 @@
+# analytics
+#
+# frontend applications are run on the ingress asg
+
+resource "aws_security_group" "analytics_task" {
+  name        = "${var.deployment}-analytics-task"
+  description = "${var.deployment}-analytics-task"
+
+  vpc_id = "${aws_vpc.hub.id}"
+}
+
+locals {
+  analytics_location_blocks = <<-LOCATIONS
+
+  set $analytics "https://analytics.tools.signin.service.gov.uk";
+  location /analytics {
+    proxy_pass $analytics/matomo.php?$args;
+  }
+
+  location /healthcheck {
+    return 200;
+  }
+
+  LOCATIONS
+
+  nginx_analytics_location_blocks_base64 = "${base64encode(local.analytics_location_blocks)}"
+}
+
+data "template_file" "analytics_task_def" {
+  template = "${file("${path.module}/files/tasks/analytics.json")}"
+
+  vars {
+    deployment             = "${var.deployment}"
+    nginx_image_identifier = "${local.tools_account_ecr_url_prefix}-verify-nginx-tls@${var.nginx_image_digest}"
+    location_blocks_base64 = "${local.nginx_analytics_location_blocks_base64}"
+  }
+}
+
+module "analytics_ecs_roles" {
+  source = "modules/ecs_iam_role_pair"
+
+  deployment       = "${var.deployment}"
+  service_name     = "analytics"
+  tools_account_id = "${var.tools_account_id}"
+}
+
+resource "aws_ecs_task_definition" "analytics" {
+  family                = "${var.deployment}-analytics"
+  container_definitions = "${data.template_file.analytics_task_def.rendered}"
+  network_mode          = "awsvpc"
+  execution_role_arn    = "${module.analytics_ecs_roles.execution_role_arn}"
+}
+
+resource "aws_security_group_rule" "analytics_task_egress_to_internet_over_http" {
+  type      = "egress"
+  protocol  = "tcp"
+  from_port = 80
+  to_port   = 80
```

Comment:
why do we need port 80 access? As far as I'm aware, the task only needs to talk over https?